travel-event-host-app/src/app/clients/event/event-client.ts

```diff
@@ -0,0 +1,37 @@
+import CategoriesType from '@/components/searchSection/types';
+import Event from '@/models/event';
+
+export async function getEventsBySearchQuery(
+  keyword: string,
+  categories: CategoriesType,
```

Comment:
Instead of passing the entire categories: boolean object, we can simply pass a string[]